Automattic/gravatar
# Update Hovercards style

## Proposed Changes

This PR switches the hovercards styles from the current wide format to a portrait design.
This new format will give users more space to display important profile elements, such as calls-to-action (CTAs), URLs, verified badges, and the new Hero image.

https://github.com/user-attachments/assets/bd4cbbdb-1647-40ac-a218-b9b228fe3d92

## Testing Instructions

* Checkout this branch and, using your terminal, navigate to `/web/packages/hovercards`
* Run `npm run start` - the playground window should open
* Play with the hovercards in the playground and check if they work as expected
* If necessary, you can change the hovercards data here:
  *  `web/packages/hovercards/playground/core.ts`
  * `web/packages/hovercards/playground/index.html`
